src/lighteval/models/vllm/vllm_model.py

```diff
@@ -66,6 +66,30 @@
 STARTING_BATCH_SIZE = 512
 
 
+@dataclass
+class VLLMModelConfig:
```

Comment:
we need to find a way to correclty parse the model args

Reply:
I'm adding it in the next PR :)